src/axolotl/core/trainer_builder.py

```diff
@@ -738,7 +738,9 @@ def build(self, total_num_steps):
             False if self.cfg.ddp else None
         )
         training_arguments_kwargs["group_by_length"] = self.cfg.group_by_length
-        training_arguments_kwargs["report_to"] = "wandb" if self.cfg.use_wandb else None
+        training_arguments_kwargs["report_to"] = (
+            "wandb" if self.cfg.use_wandb else "mlflow" if self.cfg.use_mlflow else None
+        )
```

Comment:
```suggestion
        report_to = None
        if self.cfg.use_wandb:
            report_to = "wandb"
        if self.cfg.use_mlflow:
            report_to = "mlflow"
        training_arguments_kwargs["report_to"] = report_to
```
@NanoCode012 like this?